### PR TITLE
Wrap tables and <pre> blocks to allow scrolling

### PR DESCRIPTION
**Pull request by benbc**

### CODE REVIEW

File: outputs/views.py [hunk 1]
```diff
@@ -56,18 +56,33 @@ def output_fetch_view(request, github_output):
...
+        if soup.html.body:
+            content = _contents_of_tag(soup.html.body)
         else:
-            html_content.append(content)
-    return mark_safe("".join(html_content).strip())
+            raise ValueError("HTML document has an <html>, but no <body>.")
+    else:
+        content = soup
+
+    # As well as removing any <head>, we defensively remove <script> or <style> elements that may have been inserted
```
Comment (sebbacon) [inline outputs/views.py:72]: As a possibly-unhelpful drive-by comment, there are all manner of cunning ways to embed XSS etc other than with these tags; [`lxml` has an implementation](https://lxml.de/lxmlhtml.html#cleaning-up-html) of something that may be more robust to this. If we want it.

**Review by ghickman — APPROVED** [review]
Looks good.

There's a variety of HTML sanitisation libraries available which might be of use here, I'll leave it up to you to make the call though!

**Comment by benbc** [comment]
@ghickman and @sebbacon Thank you. Right now we're mostly ripping out the scripts and styles so they don't mess rendering in the Output Explorer. For the moment we have a fixed set of reports that the tech team are handling one by one, so security isn't a big concern.

But the stuff you're gesturing at would be v helpful later on -- I'll add an issue.
